piecrust-uplink/src/abi/state.rs

```diff
@@ -19,15 +19,24 @@ use crate::{
     SCRATCH_BUF_BYTES,
 };
 
+pub enum BIndex {
+    Allowance = 0,
+    Charge = 1,
+}
```

Comment:
Maybe we could avoid making this public? Or we could make a richer type, for example:

```rust
/// Probably choose a different name for the enum 😅
pub enum EconomicChoice {
    // Different regimes that can be chosen by the contract.
    Allowance(u64),
    Charge(u64),
    None
}
```

Reply:
done